### ISSUE TITLE
Default image doesnt show in album (render_image_item statment)

### ISSUE THREAD

**Issue by ImZelleDuh** [henkelmax/camera#48]
**Please read the [FAQ](https://modrepo.de/minecraft/camera/faq) before submitting a bug report!**

**Bug description**
When editing the client config file to disable images rendering as items in-game to increase performance when opening the album setting the statement to false causes the item to not render any image instead of the default image it should be showing.

**Steps to reproduce the issue**
open camera-client visual studio code
set render_image_item = true to = false
save and open the game
look at any image in your inv or the album. it won't show its there as an item but it is there it just doesn't have a display image to show if its there (if that makes any sense) (like that trick where you give a friend a light grey glass pain but they can't see it since its the same color as the Minecraft default GUI.)

**Expected behavior**
when you look at an image item in your inv it should show the png for default_image but it doesn't display any image just a blank item with text when hovered over

**Log files**
n/a

**Versions**
all on 1.19

**Other mods**
architectury-5.7.28-forge
camera-1.19-1.0.2
cloth-config-7.0.72-forge
light-overlay-7.0.0-forge
sophisticatedbackpacks-1.19-3.17.9.617
sophisticatedcore-1.19-0.3.6.55
oculus-mc1.19-1.2.5 fork of iris for forge
rubidium-0.6.0a fork of sodium for forge

**Screenshots (Optional)**
https://imgur.com/noeEU1Z
https://imgur.com/KDzRAXH

an extra note/request. I can make this its own issue thread if you would like
is there a way to not have the image get compressed and or save the image as a png and not a jpg so I can use them for youtube thumbnails? also so when I view them in-game there isn't a loss in quality. maybe make it a config option since obv it would take up more file space and not everyone wants that. when editing the server config file and changing the compression value to compress the image as little as possible it still does compression and makes the images look way worse than what's on my screen in-game.

**Comment by ImZelleDuh** [comment]
I spent way too long digging around your code with a friend that actually understands how all this works and the closest I got or at least I think was in the file imageitemrenderer.java lines 19 - 24 but my friend said that that's prob not where the issue is. IDK the first thing about coding I just poked around as much as I could to see how far I could get on my own before consulting you. I made backups before digging around so it wouldn't hurt my game in any way obv.
